apps/cli/src/node/driver/k8s.ts

```diff
@@ -0,0 +1,64 @@
+import { execa } from "execa";
+import semver from "semver";
+
+import { NodeDriver, NodeStatus } from "./index.js";
+import { DApp } from "../database/index.js";
+
+export class K8sDriver implements NodeDriver {
+    constructor() {}
+
+    async init(): Promise<void> {
+        // check helm installed version
+        const { stdout: helmVersion } = await execa("helm", [
+            "version",
+            "--template",
+            "{{.Version}}",
+        ]);
+
+        const minimumVersion = ">=3.0.0";
+        if (!semver.satisfies(helmVersion, minimumVersion)) {
+            throw new Error(`Helm version must be ${minimumVersion}`);
+        }
+    }
+
+    async start(dapp: DApp, location: string): Promise<NodeStatus> {
+        const name = dapp.address.substring(2, 8).toLocaleLowerCase();
+        const chart = "oci://docker.io/cartesi/rollups-validator-node";
+        const httpProvider = "http://host.docker.internal:8545";
+        const wsProvider = "ws://host.docker.internal:8545";
+        const serverManager = "cartesi/server-manager:0.9.1";
+
+        const values = {
+            "dapp.blockHash": dapp.blockHash,
+            "dapp.blockNumber": dapp.blockNumber,
+            "dapp.contractAddress": dapp.address,
+            "dapp.httpProvider": httpProvider,
+            "dapp.image": serverManager,
+            "dapp.mnemonic.secretRef": "authority-owner",
+            "dapp.network": "localhost",
+            "dapp.transactionHash": dapp.transactionHash,
+            "dapp.wsProvider": wsProvider,
+            "postgresql.enabled": false,
+            "redis.enabled": false,
+        };
+
+        // TODO: serverManager.initContainers for IPFS download
+        // TODO: serverManager.extraVolumes for cartesi machine
+
+        const set = Object.entries(values)
+            .map(([k, v]) => `${k}=${v}`)
+            .join(",");
+        const flags = ["--set", set];
+        const cmd = await execa("helm", ["install", name, chart, ...flags], {
```

Comment:
```suggestion
        const cmd = await execa("helm", ["upgrade", "--install",name, chart, ...flags], {
```

This way, helm will upgrade existing charts and install them if a chart with that name doesn't exist.